extensions/observability-devservices/testcontainers/src/main/java/io/quarkus/observability/testcontainers/ObservabilityContainer.java

```diff
@@ -49,24 +51,19 @@ protected byte[] getResourceAsBytes(String resource) {
 
     @SuppressWarnings("OctalInteger")
     protected void addFileToContainer(byte[] content, String pathInContainer) {
-        logger().info("Content [{}]: \n{}", pathInContainer, new String(content, StandardCharsets.UTF_8));
```

Comment:
Not sure you are getting the right logger, that's why you have missing lines.

Reply:
Nah, it was that StartupLogCompressor that needed tweaking.